saleor/settings.py

```diff
@@ -57,7 +66,7 @@
 ]
 
 # Make this unique, and don't share it with anybody.
-SECRET_KEY = '{{ secret_key }}'
+SECRET_KEY = os.environ.get('SECRET_KEY')
```

Comment:
How about `os.environ.get('SECRET_KEY', '{{ secret_key }}')`? This way using this as a Django project template will continue to work.